doc/nrf/libraries/networking/aws_iot.rst

```diff
@@ -62,8 +62,8 @@ To establish a connection to the AWS IoT message broker, set the following optio
 
 To configure the required library options, complete the following steps:
 
-1. In the `AWS IoT console`_, navigate to :guilabel:`IoT core` > :guilabel:`Manage` > :guilabel:`things` and click on the entry for the *thing* created during the steps of :ref:`creating_a_thing_in_AWS_IoT`.
-#. Navigate to :guilabel:`interact`, find the ``Rest API Endpoint`` and set the configurable option :kconfig:`CONFIG_AWS_IOT_BROKER_HOST_NAME` to this address string.
+1. In the `AWS IoT console`_, navigate to :guilabel:`IoT core` > :guilabel:`Settings`.
+#. Find the ``Rest API Endpoint`` and set the configurable option :kconfig:`CONFIG_AWS_IOT_BROKER_HOST_NAME` to this address string.
```

Comment:
From the picture it looks like this should be `Device data endpoint` instead of `REST API Endpoint`, but it should probably be obvious to the user anyway what to look for

Reply:
Yes, that's the header of the section that contains the endpoint, but it's the endpoint address at the end of that section that's relevant. Maybe it's better to just say `Endpoint`?

Reply:
Ah, I see. Yeah, maybe better to use `Endoint`, but not a big deal